int32 id       # Track ID
float32 speed  # Estimated speed (e.g., pixels/second)
